index.html

```diff
@@ -33,9 +36,149 @@
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>    
+           
+        </ul>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+          
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+       
+            <ul class="tagcontainer">
+         <li class="label-project">Ruby on Rails</li>
+         <li class="label-project">Javascript</li>
+         <li class="label-project">CSS</li>
+         <li class="label-project">HTML</li>
+            </ul>
+        
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+          
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+   
+        <div class="about-me">
+            
+              <!-- <img class="right" src="./right.png" alt="image">   -->
+            <h2 class="about">About Me</h2>
+            
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+               
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
```

Comment:
- [ ] You have done a great job so far. :+1: :100:  But I think there is still room for improvement here. Just like the previous reviewer suggested, Your about section doesn't fully match the given design, You are missing the background image, I have made a demo video on how you could get those backgrounds from Figma. :+1: :smile: 

- DEMO VIDEO:

https://user-images.githubusercontent.com/107632780/235316738-0f1a37c9-4da6-41c9-9d04-3f4e8d95c4a4.mp4

Reply:
Hi @V-Blaze,
thanks for video my problem is not that. i am unable to insert the image.

Reply:
> Hi @V-Blaze, thanks for video my problem is not that. i am unable to insert the image.

Hello.. you can add the images as background image in your css ..